### PR TITLE
Update case to use bridge CNI due to limitation in OCPBUGS-2665

### PR DESCRIPTION
**Pull request by weliang1**
Currently test case use MACVLAN bridge CNI which cause https://issues.redhat.com/browse/OCPBUGS-2665, update case to use bridge CNI for sdn2ovn migration testing.

Test log:
upgrade-prepare:
https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/Runner-v3-smoke/5163/console
Run: ovn-sdn-migration-commands.sh
upgrade-check:
https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/Runner-v3-smoke/5164/console

@openshift/team-sdn-qe PTAL

### CODE REVIEW

File: features/upgrade/sdn/multus-upgrade.feature [hunk 1]
```diff
@@ -18,63 +17,50 @@
...
     # Check created pod has correct ip address on interface net1
     When I execute on the pod:
       | ip | a |
     Then the output should match:
-      | 192.168.22.10[12] |
-
+      | 22.2.2.22 |
     # Delete the created project from testing cluster
-    Given the "multus-upgrade" project is deleted
+    Given the "multus-upgrade" project is deleted
```
Comment (asood-rh) [inline features/upgrade/sdn/multus-upgrade.feature:66]: @weliang1 You may want to add a newline after this line.

Comment (anuragthehatter) [reply features/upgrade/sdn/multus-upgrade.feature:66]: Looks good except its recommended to not intentionally force delete the namespace in upgrade cases to help improve debugging on shared env.

Comment (weliang1) [reply features/upgrade/sdn/multus-upgrade.feature]: @asood-rh Fixed.

Comment (weliang1) [reply features/upgrade/sdn/multus-upgrade.feature]: > Looks good except its recommended to not intentionally force delete the namespace in upgrade cases to help improve debugging on shared env.

@anuragthehatter Update to keep the project for the debugging.

**Comment by anuragthehatter** [comment]
Thank you @weliang1 

/lgtm
/approve

**Pull request merged**